orgbee/src/app/post-dialog/post-dialog.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="dialog-content" [ngStyle]="{ 'background-color': data.backgroundColor }">
```

Comment:
Dont use ngStyle. Kindly write your styling in the css file.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0